test/package/esm/test-import.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+import '@okta/okta-signin-widget/polyfill';
+import OktaSignIn from '@okta/okta-signin-widget'; // default export
+import { OktaSignIn as NamedExport } from '@okta/okta-signin-widget'; // named export
+import packageJSON from '@okta/okta-signin-widget/package.json';
```

Comment:
Does this mean package.json become a public contract?

Reply:
Yes we are asserting that we support importing package.json file.